addon/components/o-s-s/banner.hbs

```diff
@@ -1,18 +1,20 @@
-<div class="upf-banner {{if @plain "background-color-gray-50" "background-color-white"}} {{if @selected "upf-banner--selected"}}
-            {{if @disabled "upf-banner--disabled"}} fx-1 fx-row padding-px-18 fx-xalign-center fx-gap-px-12" ...attributes>
+<div class="upf-banner fx-1 fx-row padding-px-18 fx-xalign-center fx-gap-px-12 {{this.modifierClasses}}"
+            ...attributes>
   {{#if (has-block "custom-icon")}}
     <div>{{yield to="custom-icon"}}</div>
   {{else if @icon}}
     <OSS::Icon @style={{fa-icon-style @icon}} @icon={{fa-icon-value @icon}}
-               class="upf-badge upf-badge--shape-round upf-badge--size-md" />
+               class="upf-badge upf-badge--shape-round {{this.iconSizeClass}}" />
   {{else if @image}}
     <img class="upf-badge upf-badge--size-md upf-badge--shape-round" src={{@image}} alt="banner" />
   {{/if}}
-  <div class="fx-col fx-1 fx-gap-px-3">
+  <div class="fx-col fx-1 {{if (not-eq @size 'sm') 'fx-gap-px-3'}}">
```

Comment:
Why remove the `gap-px-3` when the size is `sm`?

Reply:
because it's ugly :P

Reply:
without gap
![Screenshot 2023-11-28 at 14 30 25](https://github.com/upfluence/oss-components/assets/5032005/35cff7bb-394f-42e1-bc44-8f516dd66173)

with gap
![Screenshot 2023-11-28 at 14 30 40](https://github.com/upfluence/oss-components/assets/5032005/d5f3858b-ea48-4871-a275-c778513d3166)